packages/connector/src/index.ts

```diff
@@ -68,6 +68,10 @@ class ControllerConnector extends Connector {
     return await this.controller.delegateAccount();
   }
 
+  async openSetDelegate(address: string) {
```

Comment:
i think this could just be `setDelegate`, since if the session token is authorized, we could set it without the modal (in the future)